_includes/releases/v22.2/v22.2.0-alpha.4.md

```diff
@@ -0,0 +1,100 @@
+## v22.2.0-alpha.4
+
+Release Date: September 22, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-0-alpha-4-enterprise-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Changefeeds are more efficient during initial scan and backfill. The impact on runtime garbage-collection is significantly reduced, resulting in significantly reduced impact of the changefeed on foreground SQL latency. [#87796][#87796]
+- Backup and restore now can back up and restore user-defined function descriptors at database and cluster level. [#88023][#88023]
+
+<h3 id="v22-2-0-alpha-4-sql-language-changes">SQL language changes</h3>
+
+- A hint is now provided when importing from a CSV file fails because a null value has leading or trailing whitespace or because a value is quoted when the `allow_quoted_null` option is not set. [#87443][#87443]
+- Instead of always recommending to replace the first existing index with the same explicit columns, the index recommendation now considers all existing indexes in the table to decide the best one to replace. [#87174][#87174]
+- Owners of a backup schedule can now control their schedule using the supported pause, resume, drop, and alter queries. [#87600][#87600]
+- Previously, a schema could be renamed even when a child table was referenced by a view or UDF. Renaming the schema breaks the view or UDF since tables are referenced by name in views and UDFs. A check has been added that disallows renaming a schema that is referenced in views or UDFs. [#87540][#87540]
+- Users may now be granted the `CHANGEFEED` privilege on a table, allowing them to create changefeeds for that table even if they don't have the `CONTROLCHANGEFEED` role option or the `SELECT` privilege. Note that this still in effect gives them the ability to read the data in the table. Users with the `CONTROLCHANGEFEED` role option still need `SELECT` on each table, even if they also have `CHANGEFEED`. [#87887][#87887]
+
+<h3 id="v22-2-0-alpha-4-command-line-changes">Command-line changes</h3>
+
+- The `\c` metacommand no longer shows the password in cleartext. [#87538][#87538]
+- `cockroach demo` no longer runs multi-tenant mode by default. [#87810][#87810]
```

Comment:
I remembered that we aren't supposed to talk about `multi-tenant` mode in release notes. Perhaps should remove this.

Reply:
I wondered about that, but I also wondered if this was more about `cockroach demo` having surprising behavior for a bit and not having it anymore. I'll find out whether I should remove it or not.

Reply:
Confirmed it can be removed.